assets/js/directives/contextual-message.directive.js

```diff
@@ -37,10 +37,13 @@ function contextualMessage($cookies, $window, Wallet, SecurityCenter, filterFilt
     };
 
     scope.shouldShow = () => {
-      let balance   = Wallet.total('');
-      let security  = SecurityCenter.security.score;
-      let isTime    = scope.msgCookie ? Date.now() > scope.msgCookie.when : true;
-      return balance > 20000000 && security < 0.5 && isTime;
+      let isTime = scope.msgCookie ? Date.now() > scope.msgCookie.when : true;
+      let hasBalance    = Wallet.total('') > 0;
```

Comment:
Not that this can be null. Probably should make a test case for that just in case we change the rules and forget about the null case.